adapta/security/clients/aws/_aws_client.py

```diff
@@ -62,16 +70,21 @@ def get_credentials(self):
 
     def get_access_token(self, scope: Optional[str] = None) -> str:
         """
-         Not used in AWS.
-        :return:
+        Not used in AWS.
         """
-        raise NotImplementedError("Authentication with temporary credentials is not supported yet in AwsClient")
 
     def connect_storage(self, path: DataPath, set_env: bool = False) -> Optional[Dict]:
         """
-         Not used in AWS.
-        :return:
+        Configures the necessary storage options to be used to connect the AWS client for Delta Lake operations.
+        :return: All need storage options to set up Delta Lake storage client.
         """
+        storage_options = {
+            "AWS_ACCESS_KEY_ID": self._credentials.access_key_id,
+            "AWS_SECRET_ACCESS_KEY": self._credentials.access_key,
+            "AWS_REGION": self._credentials.region,
+            "AWS_ENDPOINT_URL": "" if self._credentials.endpoint is None else self._credentials.endpoint,
+        }
+        return storage_options
```

Comment:
```suggestion
        return {
            "AWS_ACCESS_KEY_ID": self._credentials.access_key_id,
            "AWS_SECRET_ACCESS_KEY": self._credentials.access_key,
            "AWS_REGION": self._credentials.region,
            "AWS_ENDPOINT_URL": "" if self._credentials.endpoint is None else self._credentials.endpoint,
        }
```
Do not waste time on stack allocations when you are going to return right away :)

Reply:
Thank you for the suggestion and input 🙂